random 2D array

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
```

Comment:
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
 
enum { MAXROW = 2, MAXCOL = 3 };
 
int main()
{
  
   int ix,ix2 = 0;
   int iy,iy2 = 0;
 
   char array1[MAXROW][MAXCOL];
   //char array2[MAXROW][MAXCOL];
  
   srandom(time(NULL));
 
   for (ix = 0; ix < MAXROW; ix++)
   {
       for (iy = 0; iy < MAXCOL; iy++)
       {
           array1[ix][iy] = random() % 26 + 'A';
           printf("%c ", array1[ix][iy]);
       }
       printf("\n");
   }
   printf("\n\n");
 
}